### DISCUSSION TITLE
Copy recognized songs to clipboard

### DISCUSSION THREAD

**Discussion by mark0100** [marin-m/SongRec#30] [Ideas]
I think it would be nice if you could copy one or multiple records (multiple selection) from the Recognition History to the clipboard.

This feature is not available in Songrec on Linux. I'm not sure if it is available in Songrec for Windows though.

**Comment by mark0100** [comment]
Hi Marin,

I just got a new version of SongRec with the Ubuntu updates and I noticed that you added the possibility of copying recognised songs to the clipboard.
Thanks for your effort!
Kind regards,
Mark

**Comment by mark0100** [comment]
Hi Marin,

I noticed that you can only select and copy a single recognized song at a time to the clipboard.
Is it possible to add that you can also select a block of records (with ShiftKey+LeftMouseclick) or select specific records (with CtrlKey+LeftMouseClick) and then copy them to clipboard in one batch?

This is called a multiple select feature, but i do not know if this is easy to program in Rust.

It would also be great if you could copy the selection to the clipboard with Ctrl+C.  

This would be very useful features if you make a playlist for instance with SongRec. Multiple songs are played and recognized with Songrec and then selected and copied to the clipboard in a single batch.

Thanks,

Mark.

**Comment by port19x** [comment]
Hey @mark0100 

Have you considered using pipes for this?

The following command works for me
`songrec recognize -d default | xclip -selection c`
